### ISSUE TITLE
Bioconda recipe?

### ISSUE THREAD

**Issue by apeltzer** [pontussk/PMDtools#2]
Hi Pontus!

would it be possible for you to merge in the Pull Request from Matthew Mah to support Python 3.X and make a release of PMDTools here on Github? 

That would make it possible to package PMDTools e.g. for bioconda (https://bioconda.github.io/) and thus make it available for many more users. I'd be happy to package it, but a release would be required for that.

Thanks a lot!

**Comment by apeltzer** [comment]
If you don't have the time to do this, I'd be also happy to change it accordingly here if you give me write permissions / add me as a collaborator. Don't want anything for it (not even a mention), but would like to use the tool in some of my work and thus I'm very keen to get this up and running soon :-)

**Comment by apeltzer** [comment]
PR https://github.com/pontussk/PMDtools/pull/3 should fix the Python2/Python3 compatibility. Once this is merged, theres just a release missing and I could make a conda recipe!

**Comment by apeltzer** [comment]
Thanks! I made a recipe for bioconda - will send another PR for the installation badge on the readme. 

```
conda install pmdtools=0.60
```
should install pmdtools in a conda environment for many users in a few minutes :-) 

https://github.com/bioconda/bioconda-recipes/pull/9405
